### ISSUE TITLE
Some Scale Tick options not working

### ISSUE THREAD

**Issue by jac0bbennett** [ipax77/pax.BlazorChartJs#7]
`Display` option for hiding ticks works, but `StepSize`, `Min`, `Max`, `MaxTickLimit` don't seem to work. 

I'm trying to make a bar chart with a large StepSize. I may just be missing something obvious. If it matters, I'm using `IndexAxis = "y"`.

```
Scales = new ChartJsOptionsScales
                {
                    X = new LinearAxis()
                    {
                        Ticks = new LinearAxisTick
                        {
                            Display = true,
                            StepSize = 100000
                        },
                        Grid = new Dictionary<string, object>
                        {
                            { "display", false },
                            { "drawBorder", false }
                        }
                    },
                    Y = new CartesianAxis
                    {
                        Stacked = "true",
                        Display = false
                    }
                },
```

Thanks!

**Comment by ipax77** [comment]
There was a problem with the Ticks-JsonConverter which is fixed in v0.3.4. Also `Stacked = true` and `Grid = new ChartJsGrid() ..` is possible , now.
I added a sample for a [StackedChart](https://github.com/ipax77/pax.BlazorChartJs/blob/master/src/pax.BlazorChartJs.sample/Pages/StackedChartPage.razor)
Thank you for reporting!

**Comment by jac0bbennett** [comment]
Thank you! Working great now
